blog/pymc3_vs_pymc_v4.ipynb

```diff
@@ -6,44 +6,47 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            sigma_a = pm.HalfNormal("sigma_a", 5.0)
</pre><p>This whole thing is so dense, add some blank lines between logical sections</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>